src/snapred/backend/dao/normalization/NormalizationRecord.py

```diff
@@ -33,6 +33,7 @@ class NormalizationRecord(Record, extra="ignore"):
     workspaceNames: List[WorkspaceName] = []
     calibrationVersionUsed: int = VERSION_DEFAULT
     crystalDBounds: Limit[float]
+    normalizationCalibrantSamplePath: str
```

Comment:
I'm tempted to suggest, that because this is a `NormalizationRecord`, the "normalization" part of `normalizationCalibrantSamplePath` is redundant.  And that to clarify the usage, the "normalization" only be added at the `ReductionRecord.normalizationCalibrantSamplePath`.